index.js

```diff
@@ -11,14 +12,40 @@ const nextButton = document.querySelector('[data-js="button-next"]');
 const pagination = document.querySelector('[data-js="pagination"]');
 
 // States
-const maxPage = 1;
-const page = 1;
+const maxPage = 42;
+let page = 1;
 const searchQuery = "";
 
+fetchCharacters("character", page);
+
+// Pagination
+
+nextButton.addEventListener("click", async () => {
+  page++;
+  if (page === 43) {
+    pagination.textContent = `1 / ${maxPage}`;
+  } else pagination.textContent = `${page} / ${maxPage}`;
+  if (page > 42) {
+    page = 1;
+  }
+  const data = await fetchCharacters("character", page);
+});
+
+prevButton.addEventListener("click", async () => {
+  page--;
+  if (page === 0) {
+    pagination.textContent = `${maxPage} / ${maxPage}`;
+  } else pagination.textContent = `${page} / ${maxPage}`;
+  if (page < 1) {
+    page = 42;
+  }
+  const data = await fetchCharacters("character", page);
+});
```

Comment:
See my previous comment.

The logic you will need for your `prevButton` is:

If `page` is bigger than 1, always subtract 1 from it.